src/projections/sorting.js

```diff
@@ -13,13 +13,25 @@ export default function sorting({
           sorting: {
             $td = null,
             onSort = () => {},
-            reducer = (s, { key }) => (s === key ? null : key),
-            isSorting = ({ key }) => state === key,
+            reducer = (s, { key }) => {
```

Comment:
Do we really want the default reducer to have this complex behavior?

Reply:
If we want to provide the complex behavior, I'd rather to provide helper functions. Like

```js
import bidirectionalSortStrategy from '...';
{
  ...,
  sorting: bidirectionalSortStrategy({
    ...,
  }),
},
```